zoo-css-challenge/style.css

```
@@ -105,25 +110,44 @@ footer {
 
 .image-container img {
   vertical-align: top;
-  border: 5px solid #747c92;
+  border: 5px solid #3660d3;
 }
 
+.image-container img:nth-child(2) {
+  vertical-align: top;
+  border: 5px solid #dd2626;
+}
 .image-container img:first-of-type {
-  vertical-align: bottom;
+  vertical-align: top;
+  border: 5px solid #abdf88;
+
+
 }
 
 .tigers {
   background-color: #0b3c49;
   color: #f0e100;
 }
+.tigers h2, h3{
```

```suggestion
.tigers h2, .tigers h3{
```
The `h3` selector will change the colour for all `h3`s on the page, but the task is to only change it for the tigers section so we should update the selector to `.tigers h3`